docs/ossm/ossm2-migration/network-policies.md

```diff
@@ -0,0 +1,329 @@
+            matchLabels:
+              service-mesh: enabled
+  podSelector: {}
+  policyTypes:
+    - Ingress
+```
+
+Expose Route Network Policy for httpbin-2 namespace
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-expose-route-basic
+  namespace: httpbin-2
+spec:
+  podSelector:
+    matchLabels:
+      maistra.io/expose-route: "true"
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              network.openshift.io/policy-group: ingress
+  policyTypes:
+    - Ingress
+```
+
+Mesh Network Policy for httpbin-2 namespace
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-mesh-basic
+  namespace: httpbin-2
+spec:
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              service-mesh: enabled
+  podSelector: {}
+  policyTypes:
+    - Ingress
+```
+
+Expose Route Network Policy for httpbin-3 namespace
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-expose-route-v3
+  namespace: httpbin-3
+spec:
+  podSelector:
+    matchLabels:
+      maistra.io/expose-route: "true"
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              network.openshift.io/policy-group: ingress
+  policyTypes:
+    - Ingress
+```
+
+Mesh Network Policy for httpbin-3 namespace
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-mesh-v3
+  namespace: httpbin-3
+spec:
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchLabels:
+              service-mesh: enabled
+  podSelector: {}
+  policyTypes:
+    - Ingress
+```
+
+Note that the user re-labeled the namespaces in the mesh with label `service-mesh: enabled` and included that in the network policies accordingly, because the label `maistra.io/member-of: istio-system` will be removed from the namespaces during migration.
+
+> **_NOTE:_** We recommend that rather than use a "one size fits all" label like `service-mesh=enabled`, you use a label scoped only to this mesh that can be reused for discovery selectors. This avoids additional namespace labeling. See [discoverySelector documentation](./../create-mesh/README.md) for more details.
+
+Now the user has disabled the `manageNetworkPolicy` setting while following along the pre-migration steps and is about to create the 3.0 mesh which will be named `v3` and created in `istio-system`.
+
+Before creating the `istio` resource and the istiod pod, the user makes:
+
+Istiod Network Policy for 3.0:
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: istio-istiod-v3
+  namespace: istio-system
+spec:
+  ingress:
+    - {}
+  podSelector:
+    matchLabels:
+      app: istiod
+      istio.io/rev: v3
+  policyTypes:
+    - Ingress
+```
+
+to ensure that they are consistent in Network Policies with the 3.0 mesh before continuing on with the migration steps. 
```

Comment:
I think this section would be more clear if it were steps. The first step, picking and re-labeling your namespaces, is in the middle of this section as a `NOTE:`.

From what I can see the steps are:
1. Add new label(s) to namespaces
2. Create network policies with the new labels
3. Disable `manageNetworkPolicy`

Reply:
Good point, made this change https://github.com/openshift-service-mesh/sail-operator/pull/203/commits/89653b17d4861502a42a2d9835c395a5601e3bde